Add codebooks for sdg 2, 3 and 4 (#19)
nestauk/sdg-mapping: codebooks/codebook_sdg_4.md

# Goal 4. Quality Education
**Ensure inclusive and equitable quality education and promote lifelong learning opportunities for all**

## Labelling Instructions

### Key Concepts of Goal 4

> To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of:

Reviewer: ```suggestion
```

> - Ensure all children have access to quality and free pre-primary childhood development, care and education.

Reviewer: ```suggestion
- Quality and free, equitable and quality primary and secondary education
- Quality and free pre-primary childhood development, care and education
- Access to tertiary education and university

```

> - Equal access for men and women affordable vocational, technical and tertiary education

Reviewer: ```suggestion
- Equal access for men and women to vocational, technical and tertiary education

```

- Eliminate gender disparities in access to education and vocational training and for the vulnerable*

> - Increase in number of youth and adults who have relevant skills for employment and entrepreneurship

Reviewer: ```suggestion
- Relevant skills for employment and entrepreneurship for youth and adults

```

> - Achieve literacy and numeracy amongst youth and some adults

Reviewer: ```suggestion
- Literacy and numeracy for youth and adults

```

> - Leaners to acquire knowledge and skills needed to promote sustainable development such as human rights, gender equality, promotion of cultural diversity, etc*  

Reviewer: ```suggestion
- Learning knowledge and skills to promote sustainable development such as human rights, gender equality, promotion of cultural diversity, etc* 

```

> - increase the supply of qualified teachers and scholarships available to developing countries

Reviewer: ```suggestion
- Supply of qualified teachers and scholarships available to developing countries
- Education facilities that are safe and child, disability and gender sensitive
```

To be labelled with `Yes`, a project should explicitly or implicitly refer to at least one of these. More detail on the specific targets is provided below.

* There are specific goals for gender, reducing inequalities and gender equality. Please only label projects `Yes` if they mainly relate to these areas in the context of state spending.

### Key terms (non-exhaustive)
education, quality, leaning, school, university, college, academic, vocational, training, education system, literacy, curriculum, STEM, ability, e-book, books, creativity, open learning, student, teacher training, numeracy, access to education

### Not to be confused with

- 5. Gender Equality

>   - Concerns gender equality in various areas, not just education (specifically on reproductive health)

Reviewer: ```suggestion
  - Concerns broader gender equality

```

- 8. Decent Work and Economic Growth

>   - Focus on types of work conditions and improving working environments

Reviewer: ```suggestion
```

Remember, many projects will cover more than one Goal, but to be tagged as Goal 4 they must include at least one of the Key Concepts above. Further information on other goals can be found [here](https://unstats.un.org/sdgs/metadata/).

### Examples

| Project description                                                                        | Is Goal 4? |
|--------------------------------------------------------------------------------------------|------------|
| Project discussing the European infrastructure for open learning                    | Yes        |
| Project discussing the creation of a game which aims to develop and deliver a curriculum in soft skills                    | Yes        |
| Project discussing a collaboration ecosystem enabling EU creative SMEs to exchange multi-media content and create multi-plot, interactive apps for children                   | Yes        |
| Project discussing building capacity for design enabled innovation in urban environments                    | No         |

> | Project discussing improving the knowledge of organised crime                    | No         |

Reviewer: ```suggestion
| Development of non-education related games for children                    | No         |

```
